docs/tutorial/replicated-volumes.md

```diff
@@ -0,0 +1,186 @@
+# Creating Replicated Volumes
+
+Learn about creating replicated volumes, making your data accessible on any cluster node.
+
+In this tutorial you will learn how to create a replicated volume, and verify that the data stays accessible when
+moving Pods from one node to another.
+
+## Prerequisites
+
+* A Kubernetes Cluster with at least two nodes.
+* An installed and configured Piraeus Datastore. Learn how to get started in our [introduction tutorial](./get-started.md)
+
+## Creating the Volume
+
+First, we will create a new [`StorageClass`](https://kubernetes.io/docs/concepts/storage/storage-classes/) for our
+replicated volumes. We will be using the `pool1` storage pool from the [Get Started tutorial](./get-started.md), but
+this time also set the `placementCount` to 2, telling LINSTOR® to store the volume data on two nodes.
+
+```
+$ cat <<EOF | kubectl apply -f -
```

Comment:
Why not

```suggestion
$ kubectl apply -f - <<EOF
```

?

Not sure why I noticed this now.

Reply:
Excellent question. Replaced it everwhere.